OpenRA.Mods.Common/Activities/Resupply.cs

```diff
@@ -79,7 +79,8 @@ public override bool Tick(Actor self)
 			// HACK: If the activity is cancelled while we're already resupplying (or about to start resupplying),
 			// move actor outside the resupplier footprint.
 			// TODO: This check is nowhere near robust enough, and should be rewritten.
-			if (IsCanceling && host.IsInRange(self.CenterPosition, closeEnough))
+			var isCloseEnough = closeEnough.LengthSquared == 0 || (host.CenterPosition - self.CenterPosition).HorizontalLengthSquared <= closeEnough.LengthSquared;
```

Comment:
Does anyone know why the original code here used `closeEnough == 0` to mean "disable the range check", rather than `closeEnough < 0`? This removes the ability to say "I want it *exactly* at this position".

Reply:
I noticed that, too. I think I faintly remember it had something to do with aircraft, but that's from a time well before we started refactoring this. Could very well be obsolete by now.